Vungle/VungleRouter.m

```diff
@@ -503,59 +527,71 @@ - (BOOL)validateInfoData:(NSDictionary *)info
     return isValid;
 }
 
-- (void)clearDelegateWithState:(BannerRouterDelegateState)state placementID:(NSString *)placementID
```

Comment:
This method is split because it performs two different actions based on the provided parameters (see the original https://github.com/Vungle/mopub-ios-mediation/blob/6.9.x_pre_release/Vungle/VungleRouter.m#L506)

If placementID is set, cleanup the full screen delegate dictionary. If the banner state is set, cleanup the banner delegate dictionary. The method are split into 2 methods to handle each case.